cumulus/parachains/runtimes/people/people-westend/src/people.rs

```diff
@@ -0,0 +1,225 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::*;
+use codec::{Decode, Encode, MaxEncodedLen};
+use enumflags2::{bitflags, BitFlags};
+use frame_support::{
+	parameter_types, traits::ConstU32, CloneNoBound, EqNoBound, PartialEqNoBound,
+	RuntimeDebugNoBound,
+};
+use pallet_identity::{Data, IdentityInformationProvider};
+use parachains_common::impls::ToParentTreasury;
+use scale_info::{build::Variants, Path, Type, TypeInfo};
+use sp_runtime::{traits::AccountIdConversion, RuntimeDebug};
+use sp_std::prelude::*;
+
+parameter_types! {
+	//   27 | Min encoded size of `Registration`
+	// - 10 | Min encoded size of `IdentityInfo`
+	// -----|
+	//   17 | Min size without `IdentityInfo` (accounted for in byte deposit)
+	pub const BasicDeposit: Balance = deposit(1, 17);
+	pub const ByteDeposit: Balance = deposit(0, 1);
+	pub const SubAccountDeposit: Balance = deposit(1, 53);
+	pub IdentityPalletAccount: AccountId =
+		parachains_common::polkadot::account::IDENTITY_PALLET_ID.into_account_truncating();
+	pub RelayTreasuryAccount: AccountId =
+		parachains_common::polkadot::account::POLKADOT_TREASURY_PALLET_ID.into_account_truncating();
+}
+
+impl pallet_identity::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Currency = Balances;
+	type BasicDeposit = BasicDeposit;
+	type ByteDeposit = ByteDeposit;
+	type SubAccountDeposit = SubAccountDeposit;
+	type MaxSubAccounts = ConstU32<100>;
+	type IdentityInformation = IdentityInfo;
+	type MaxRegistrars = ConstU32<20>;
+	type Slashed = ToParentTreasury<RelayTreasuryAccount, IdentityPalletAccount, Runtime>;
+	type ForceOrigin = EnsureRoot<Self::AccountId>;
+	type RegistrarOrigin = EnsureRoot<Self::AccountId>;
+	type WeightInfo = weights::pallet_identity::WeightInfo<Runtime>;
+}
+
+/// The fields that we use to identify the owner of an account with. Each corresponds to a field
+/// in the `IdentityInfo` struct.
+#[bitflags]
+#[repr(u64)]
+#[derive(Clone, Copy, PartialEq, Eq, RuntimeDebug)]
+pub enum IdentityField {
+	Display,
+	Legal,
+	Web,
+	Matrix,
+	Email,
+	PgpFingerprint,
+	Image,
+	Twitter,
+	GitHub,
+	Discord,
+}
+
+impl TypeInfo for IdentityField {
```

Comment:
we probably do not need `TypeInfo` impl for `IdentityField`, I do not see where it is used.

Reply:
Indeed we don't need it anymore. It used to be an argument in an extrinsic, but now it's satisfied thanks to `IdentityInformationProvider::FieldsIdentifier` being `u64` directly for legacy structs. Good catch!